### PR TITLE
Exercise #8: 'acc' key should be 'accuracy'

### PR DESCRIPTION
**Pull request by agonzalezro**
The snippet in exercise #8 is trying to access the key 'acc' that doesn't
exist in keras.models.Sequential().fit(...).history. It should be accessing
'accuracy' instead.

### CODE REVIEW

**Comment by agonzalezro** [comment]
I am having the same problem but the other way around (instead `accuracy` it's expecting `acc`) and the model seems to be `Sequential` again so I don't understand what's going on.

Could it be using different versions of `keras`?

**Comment by agonzalezro** [comment]
Sorry for bothering again, but the last exercise (#10) has the same problem that the one shown in this PR: `acc` should be `accuracy`. So, summarising:

- this PR is for exercise #8 and expects `acc` to be `accuracy`.
- exercise #9 expects `accuracy` to be `acc`.
- and finally exercise #10 expects `acc` to be `accuracy`.

BTW, I forgot to mention that I am running the exercises in Microsoft Docs webs and not locally, so I am using the versions provided by your Jupyter notebooks.

Thanks!
